content/en/events/README.md

```diff
@@ -0,0 +1,34 @@
+This directory lets you record promotional events for the product. You create files in here and
+the site will automatically take care of displaying the promotional info about the event at the
+right time, and in the right places on the site.
+
+There are two types of promotions supported:
+
+- Countdown stickers that show how much time is left before something big happens. (e.g. "2 days until Istio 1.5").
+Stickers have minimal visuals displayed to the user during the given period.
+
+- Banners that display a prominent message to the user to let them know a significant event is about to take place,
+is taking place, or has taken place. (e.g. "Istio 1.5 has been released, download it today!" or "Join us at Kubecon on March 30").
+Banners are full-screen slices displayed to the user during the given period.
+
+You control the general behavior via front-matter fields in every event file. The fields are:
+
+- title. The name of the event. This is not displayed on the web site, it's intended for diagnostic messages.
+
+- event_trigger: Events can be triggered based on a date or based on a new release. When this field is set to `event_date`,
+then the event is triggered based on the value of the `event_date` field below. If this field is instead set to `latest_release`,
+then this event is considered to have happened at the release date of the latest Istio release. This is used to
+automatically display a banner that says "Istio X.Y.Z has just been released" whenever a new release is produced.
+
+- event_date. The date of the event in YYYY-MM-DD format.
+
+- countdown_days. How long before the event to display the item to the user.
+
+- event_days. How long after the event to display the item to the user.
+
+- visibility. How many times to show the content to the user, or the string `always` to display it on every page during
```

Comment:
I wonder if a boolean would be sufficient ... display once or always. Picking a number in between would seem hard, and might end up looking like random behavior. Anyway, if it's not hard to keep track, i guess it doesn't hurt to allow any number.